pkg/dynamicinstrumentation/module_unsupported.go

```diff
@@ -0,0 +1,19 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build !linux
+// +build !linux
+
+package dynamicinstrumentation
+
+import (
+	"github.com/DataDog/datadog-agent/cmd/system-probe/api/module"
+	"github.com/DataDog/datadog-agent/pkg/ebpf"
+)
+
+// NewModule instantiates a dynamic instrumentation system-probe module
+func NewModule(cfg *Config) (module.Module, error) {
```

Comment:
I don't think you need this file, since you only list the module in `modules/all_linux.go`

Reply:
Yes I believe you're right.